keyboard.js

```diff
@@ -0,0 +1,68 @@
+class Keyboard {
+    constructor(keySet, language) {
+
+        this.isCapsLock = false;
+        this.isShift = false;
+        this.language = language;
+        this.keys = keySet;
+        this.controlKeys = [
+            'Backspace',
+            'Tab',
+            'CapsLock',
+            'Enter',
+            'ShiftLeft',
+            'ShiftRight',
+            'ControlLeft',
+            'ControlRight',
+            'ContextMenu',
+            'AltLeft',
+            'AltRight',
+        ];
+    }
+
+    keySet() {
+        return this.keys;
+    }
+
+
+    checkControlKey(code) { /////////////////////////////
+        return this.controlKeys.some((controlKey) => controlKey === code); // some проверяет на соответствие условия заданного в передаваемой функции
+
+    }
+
+    checkCapsLock() {
+        return this.isCapsLock;
+    }
+
+    toggleCapsLock() {
+        this.isCapsLock = !this.isCapsLock;
+    }
+
+
+    checkShift() {
+        return this.isShift;
+    }
+
+    toggleShift() {
+        this.isShift = !this.isShift;
+    }
+
+    checkLanguage() {
+        return this.language;
+    }
+
+    toggleLanguage() {
+        if (this.language === 'en') {
```

Comment:
it would be nice to create constant for language:

```js
const LANGUAGE = {
  EN: 'en',
  RU: 'ru',
};
```

then in your code you can use LANGUAGE.EN or LANGUAGE.RU instead of strings.